go/pkg/sysdb/coordinator/coordinator.go

```diff
@@ -229,6 +233,10 @@ func (s *Coordinator) GetTenantsLastCompactionTime(ctx context.Context, tenantID
 	return s.catalog.GetTenantsLastCompactionTime(ctx, tenantIDs)
 }
 
+func (s *Coordinator) SetTenantResourceName(ctx context.Context, tenantID string, resourceName string) error {
```

Comment:
Should we instead prefer having a `UpdateTenant` call ? cc @sanketkedia

Reply:
i am ok with that for now, given its the only field, but at n>2 we should change this. can you leave a comment to that effect?